modules/lib/lib-event/.eslintrc.js

```diff
@@ -0,0 +1,83 @@
+module.exports = {
```

Comment:
it should be common for all libs

Reply:
Of course. I would like to move the definitions to our own `@enonic/eslint-recommended` package and use it later as a plugin, so the only thing that we need to do is to write `module.exports = {'extends': ['plugin:@enonic/eslint-recommended']}`. Actually, I needed to be done long ago, since it's a pain in the back to sync all the ESLint configs across our apps. As for the file itself, I would rather keep it inside the package folder, along with the `package.json` and `tsconfig.json`.